internal/process/provisioning/create_kyma_name_step.go

```diff
@@ -0,0 +1,44 @@
+package provisioning
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/kyma-project/kyma-environment-broker/internal/process/steps"
+
+	"github.com/kyma-project/kyma-environment-broker/internal"
+	"github.com/kyma-project/kyma-environment-broker/internal/process"
+	"github.com/kyma-project/kyma-environment-broker/internal/storage"
+	"github.com/sirupsen/logrus"
+)
+
+type CreateKymaNameStep struct {
+	operationManager *process.OperationManager
+}
+
+func NewCreateKymaNameStep(os storage.Operations) *CreateKymaNameStep {
+	return &CreateKymaNameStep{
+		operationManager: process.NewOperationManager(os),
+	}
+}
+
+func (s *CreateKymaNameStep) Name() string {
+	return "Create_Kyma_Name"
+}
+
+func (s *CreateKymaNameStep) Run(operation internal.Operation, log logrus.FieldLogger) (internal.Operation, time.Duration, error) {
+	if time.Since(operation.UpdatedAt) > CreateRuntimeTimeout {
```

Comment:
In my opinion, we do not need to check any timeout

Reply:
Removed in the next commit.